### PR TITLE
CLN: f-string in pandas/core/window/*

### PR DESCRIPTION
**Pull request by ganevgv**
ref #29547

- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by ganevgv** [comment]
finally ready for review

File: pandas/core/window/expanding.py [hunk 1]
```diff
@@ -209,10 +209,10 @@ def skew(self, **kwargs):
 
     >>> arr = [1, 2, 3, 4, 999]
     >>> import scipy.stats
-    >>> fmt = "{0:.6f}"  # limit the printed precision to 6 digits
-    >>> print(fmt.format(scipy.stats.kurtosis(arr[:-1], bias=False)))
+    >>> precision = 6  # limit the printed precision to 6 digits
```
Comment (WillAyd) [inline pandas/core/window/expanding.py]: Minor but I find this rather distracting from the code sample. Can you just hard-code the format specifier in the print statements below?

**Review by WillAyd — CHANGES_REQUESTED** [review]

File: pandas/core/window/expanding.py (hunk 1 above)
Comment (ganevgv) [reply pandas/core/window/expanding.py]: Sounds good - just did that.

**Review by WillAyd — APPROVED** [review]
lgtm

**Comment by jreback** [comment]
thanks @ganevgv

**Pull request merged**